src/elements/teaser-hero/readme.md

```diff
@@ -0,0 +1,55 @@
+The `sbb-teaser-hero` is a component with a background image and an action call within a panel;
```

Comment:
I think it should somehow be included in docs, how the chip only variant can be achieved and generally, what can be done with the chip